src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -3271,7 +3271,7 @@
     <value>Files is running as administrator</value>
   </data>
   <data name="FilesRunningAsAdminContent" xml:space="preserve">
-    <value>Due to platform limitations, drag and drop isn't available when running Files as administrator.</value>
+    <value>Due to a limiation with Windows and WinAppSdk, drag and drop isn't available when running Files as admin. If you wish to use drag and drop, you can work around this limitation by opening UAC (User Account Control) from the Start Menu and selecting the third level, and restarting Windows. Otherwise, you can keep using Files without drag &amp; drop.</value>
```

Comment:
Most users don't know what WinAppSdk is.

SDK is better I believe.

Reply:
Or SDK is also part of Windows so even might not needed.

Reply:
We might want to add some line breaks as well, I'll have a better idea once I compile this PR on my device.

Reply:
I agree that WinAppSDK part is not needed and would be less to read. I would also suggest that the instructions are formatted as to make it more clear
1.
2.
3.

Reply:
MarkdownTextBlock might help you